doc/development-guide.md

```diff
@@ -0,0 +1,2 @@
+# Development Guide
```

Comment:
I would just leave out this file if it has no content.

Reply:
Or put a TODO line saying we will write this in a while, or anyone interested in contributing to doc can write it up.